packages/components/nodes/memory/ZepMemoryCloud/ZepMemoryCloud.ts

```diff
@@ -0,0 +1,181 @@
+import { IMessage, INode, INodeData, INodeParams, MemoryMethods, MessageType } from '../../../src/Interface'
```

Comment:
I think this would be doable, but I have some reservations. 

Our interfaces are not the same for open source and cloud sdk. For memory, for example, we don't offer the ability to specify `memoryType` on Open Source version, whereas this is one of the new things we have introduced with Zep Cloud. The same applies to vector stores, in cloud fro example we don't accept embeddings and embedding dimensions any more.

Reply:
So we would have to have a lot of additional if blocks in the nodes.

Reply:
Oh okay i thought the difference is only the initialization. In that case, make sense to have separate nodes

Reply:
Cool! Marked it as ready for review then!